configuration/config-txt/overclocking.md

```diff
@@ -12,7 +12,7 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 | --- | --- |
 | arm_freq | Frequency of the ARM CPU in MHz; default value is `1000` for the Pi Zero and Pi Zero W, `700` for Pi 1, `900` for Pi 2, `1200` for the Pi 3, `1400` for the Pi 3B+, `1500` for the Pi 4B|
 | gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 1/Pi 2, the default value is `250` for all items, on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on the Pi 4B the `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` default to `500`, `600` is the only other accepted value|
-| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for the Pi 1/Pi 2, `400` for the Pi 3/Pi Zero/Pi Zero W. For the Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only the Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on the Pi 2/Pi 3, and there is no effect on the SDRAM on the Pi 4B|
+| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for the Pi 1/Pi 2 and `400` for the Pi 3/Pi Zero/Pi Zero W. For the Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only the Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on the Pi 2/Pi 3, and there is no effect on the SDRAM on the Pi 4B|
```

Comment:
This isn't correct. core_freq affects sdram bandwidth on a Pi4 just like the Pi2/Pi3.

Reply:
Fixed in the latest edit